workshop1.adoc

```diff
@@ -2,95 +2,107 @@
+* Create a new file `autocompletion.gdsl` on the `src` directory and copy the copied content
+* Create a file `pipeline.groovy` inside `src` directory and try to autocomplete things inside
+* IntelliJ IDEA should help you with autocompletion of methods available on scripted Jenkins Pipeline (Ctrl+Space)
 
-=== Création de l'ossature de notre pipeline
+TIP: IntelliJ should propose you to activate GDSL module after the `autocompletion.gdsl` file creation
 
-Le but de cette partie est de créer un pipeline "vide" (sans traitement réel, on veut juste jouer avec les capacités de Pipeline pour l'instant).
+TIP: Article about GDSL : http://www.tothenew.com/blog/gdsl-awesomeness-introduction-to-gdsl-in-intellij-idea/
 
-Il faut respecter les caractéristiques suivantes :
+=== Create our Pipeline structure
 
-* Le résultat final doit ressembler à ceci côté Blue Ocean :
+This part goal is to create and "empty" pipeline (without real build/test). We just want to play with Jenkins Pipeline capabilities for the moment.
+
+You to fulfill the following requirements :
+
+* The final result must looks like the following on Blue Ocean :
 
 image::images/workshop1-empty-pipeline.png[Jenkins Pipeline]
 
-* Entre la phase  `staging` et `deploy`, l'étape "manual-approval" permet de demander une action manuelle afin d'approuver le déploiement.
-Ajouter un `input` permettant d'approuver ou non le déploiement.
+* Between the `staging` and `deploy` stages, the step `manual-approval` permit to ask a manual action to approve the deployment.
+Add an `input` permitting to approve or not the deployment.
 
 image::images/workshop1-empty-pipeline2.png[Jenkins Pipeline]
 
-* Les stages `build & unit tests`, `static-analysis` doivent allouer un agent avec le label `build` et effectuer un `sleep` de 5s.
+* The `build & unit tests`, `static-analysis` stages must allocate and agent with the `build` label and do a `sleep` of 5s.
 
-TIP: si votre pipeline semble bloquer lorsque vous utilisez `node("build") {`, relisez attentivement l'avertissement au début de cette page.
+TODO A GARDER? TIP: si votre pipeline semble bloquer lorsque vous utilisez `node("build") {`, relisez attentivement l'avertissement au début de cette page.
 La configuration du Master n'est pas correcte par défaut, il faut corriger la configuration du plugin Digital Ocean.
 
-* Le stage `tests` doit paralléliser 3 traitements: `chrome`, `edge` et `firefox`, chacun effectuant un `sleep` de 5s sur un node différent (peu importe le label)
+* The `tests` stage must parallelize three tasks : `chrome`, `edge` and `firefox`, each one needs to do a sleep of 5s on a different node (any label)
+
+* Les stages `staging` et `deploy` must allocate any agent and do a `sleep` of 5s
 
-* Les stages `staging` et `deploy` doivent allouer un agent (n'importe lequel) et effectuer un `sleep` de 5s.
+NOTE: This exercise is voluntary not detailed.
 
+<<<<<<< f5b3325a6e374849533e4f469eaf682354335ecb
```

Comment:
conflict here, will close this PR and file another one as I don't think I have access to your fork.